### ISSUE TITLE
Auto detect URL in the proposal and highlight

### ISSUE THREAD

**Issue by kracekumar** [pythonindia/junction#539]
When a user misses using markdown syntax to highlight the URL in the proposal section, automatically highlight the URL in the text.

URL: https://in.pycon.org/cfp/2017/proposals/pytorch-for-the-tensorflow-developers-and-others-dl-enthusiasts~dLApe/

![url_missing_highlight](https://user-images.githubusercontent.com/311929/27913782-c647e902-627e-11e7-9181-e6094a457f32.png)

**Comment by kracekumar** [comment]
In the above description, I didn't use Markdown syntax, still, GH highlights the URL.

**Comment by mish24** [comment]
Taking this up

**Comment by kracekumar** [comment]
@mish24 Please go ahead.

**Comment by mish24** [comment]
@kracekumar mostly in `content_urls` there's the need to use angular brackets to identify multiple URLs and convert them. This won't recognize any link if it's already inside any block of code. Basically I was going to implement simple regex scraping for storing urls here. Shall I go ahead? In my opinion telling the users to use angular brackets seems like a much better option. What are your thoughts?

**Comment by kracekumar** [comment]
@mish24 Giving heads up the field supports the markdown is an excellent option. The form field should auto detect URLs and add respective markup. I'm sure the existing Markdown Field or another plugin can do this. Can you look around?

**Comment by mish24** [comment]
Yeah. I'll upgrade to `django-markdown2` and write some filters. That'll do it. Sending PR soon

**Comment by smtchahal** [comment]
This seemed to work fine until 408f0c0 removed the '`linkify`' markdown extension, any reason why that was done?
